### PR TITLE
Ibp 5395 customize my study entry list columns and details

### PR DESCRIPTION
**Pull request by clarysabel**

### CODE REVIEW

File: src/main/java/com/efficio/fieldbook/web/study/germplasm/StudyEntryTransformer.java [hunk 1]
```diff
@@ -50,151 +47,24 @@ public List<ImportedGermplasm> tranformToImportedGermplasm(final List<StudyEntry
...
-				final Optional<String> groupGid = studyEntryDto.getStudyEntryPropertyValue(TermId.GROUPGID.getId());
+				final Optional<Integer> groupGid =  Optional.ofNullable(studyEntryDto.getGroupGid());
 				if (groupGid.isPresent()) {
 					final Integer mgid = Integer.valueOf(groupGid.get());
 					importedGermplasm.setMgid(mgid);
 					importedGermplasm.setGroupId(mgid);
 				}
-				importedGermplasm.setSource(studyEntryDto.getStudyEntryPropertyValue(TermId.SEED_SOURCE.getId()).orElse(""));
-				importedGermplasm.setGroupName(studyEntryDto.getStudyEntryPropertyValue(TermId.CROSS.getId()).orElse(""));
+				//FIXME: The GroupName is the Preferred Name of the groupGID
```
Comment (clarysabel) [inline src/main/java/com/efficio/fieldbook/web/study/germplasm/StudyEntryTransformer.java:61]: @cuenyad anything we need to do with this FIXME in this branch? If not, please improve the comment to work on it later on

File: src/main/java/com/efficio/fieldbook/web/util/CrossesListUtil.java [hunk 2]
```diff
@@ -153,7 +152,7 @@ public List<String> getTableHeaders() {
 		tableHeaderList.add(this.getTermNameFromOntology(ColumnLabels.CROSS_MALE_GID));
 		tableHeaderList.add(this.getTermNameFromOntology(ColumnLabels.MALE_PARENT));
 		tableHeaderList.add(this.getTermNameFromOntology(ColumnLabels.BREEDING_METHOD_NAME));
-		tableHeaderList.add(this.getTermNameFromOntology(ColumnLabels.SEED_SOURCE));
+		tableHeaderList.add("GERMPLASM_ORIGIN");
```
Comment (clarysabel) [inline src/main/java/com/efficio/fieldbook/web/util/CrossesListUtil.java:155]: @cuenyad Add a ColumnLabel for GERMPLASM_ORIGIN

File: src/main/java/com/efficio/fieldbook/web/trial/controller/AdvancingController.java [hunk 3]
```diff
@@ -594,7 +594,7 @@ protected List<TableHeader> getAdvancedStudyTableHeader() {
 		tableHeaderList.add(new TableHeader(ColumnLabels.DESIGNATION.getTermNameFromOntology(this.ontologyDataManager), "desig"));
 		tableHeaderList.add(new TableHeader(ColumnLabels.PARENTAGE.getTermNameFromOntology(this.ontologyDataManager), "parentage"));
 		tableHeaderList.add(new TableHeader(ColumnLabels.GID.getTermNameFromOntology(this.ontologyDataManager), "gid"));
-		tableHeaderList.add(new TableHeader(ColumnLabels.SEED_SOURCE.getTermNameFromOntology(this.ontologyDataManager), "source"));
+		tableHeaderList.add(new TableHeader("GERMPLASM_ORIGIN", "source"));
```
Comment (clarysabel) [inline src/main/java/com/efficio/fieldbook/web/trial/controller/AdvancingController.java:597]: @cuenyad Add a ColumnLabel for GERMPLASM_ORIGIN with null TermId

File: src/test/java/com/efficio/fieldbook/web/util/WorkbookUtilTest.java [hunk 4]
```diff
@@ -122,7 +122,8 @@ public void testFindMeasurementVariableByName() {
 		Assert.assertEquals(measurementVariable1, result.get());
 	}
 
-	@Test
+	// TODO: Commented broken tests.
```
Comment (clarysabel) [inline src/test/java/com/efficio/fieldbook/web/util/WorkbookUtilTest.java:125]: @cuenyad please work on this tests

Comment (cuenyad) [reply src/test/java/com/efficio/fieldbook/web/util/WorkbookUtilTest.java:125]: done!

File: src/main/java/com/efficio/fieldbook/web/trial/controller/AdvancingController.java (hunk 3 above)
Comment (cuenyad) [reply src/main/java/com/efficio/fieldbook/web/trial/controller/AdvancingController.java:597]: done!.

File: src/main/java/com/efficio/fieldbook/web/util/CrossesListUtil.java (hunk 2 above)
Comment (cuenyad) [reply src/main/java/com/efficio/fieldbook/web/util/CrossesListUtil.java:155]: done!.

File: src/main/java/com/efficio/etl/service/ETLService.java [hunk 5]
```diff
@@ -164,4 +164,5 @@ Map<String, List<Message>> checkForMismatchedHeaders(List<String> fileHeaders, L
    */
   String getExperimentalDesignValueFromObservationSheet(Workbook workbook, UserSelection userSelection, final int columnIndex);
 
+  Map<String, Integer> retrieveAvailableCategoricalScales(final Integer termId, final String programUUID);
```
[the remark this answers is not in the document]
Comment (cuenyad) [reply src/main/java/com/efficio/etl/service/ETLService.java:167]: done!.

File: src/main/java/com/efficio/fieldbook/web/study/germplasm/StudyEntryTransformer.java (hunk 1 above)
Comment (cuenyad) [reply src/main/java/com/efficio/fieldbook/web/study/germplasm/StudyEntryTransformer.java]: done!.

**Pull request merged**